backend/src/main/java/codezap/template/domain/Template.java

```diff
@@ -82,4 +86,15 @@ public boolean matchMember(Member member) {
     public boolean isPrivate() {
         return visibility == Visibility.PRIVATE;
     }
+
+    public void updateLike() {
```

Comment:
호옥시 `updateLike()` setter 처럼 주어진 값으로 업데이트된다는 느낌이 있는 것 같아서...
`like()` 메서드명은 어떠신가요?
```suggestion
    public void like() {
```

Reply:
음 저는 like라는 단어가 도메인 이름과도 거의 너무 비슷해서 아예 리소스로 보고 있어요 행위랑 섞이면 헷갈릴 것 같은데 짱수 생각은 어떤가요~~

Reply:
메서드명은 동사로 시작하기 때문에 전 불퍈하진 않을 것 같아요. 하지만 실제로 Likes가 있긴 하니 헷갈릴 것 같다면 다른 네이밍이 좋긴 하겠네요.
그렇다면 update 대신 increase-로 가던가..?
값을 "반영"한다는 느낌때문에 코멘트를 남긴거라서 수정되면 좋고, 안되도 큰 문제는 없을 것 같다 정도임다!

Reply:
increase 괜찮네요 굿 반영할게요~

Reply:
아싸~~